adafruit_displayio_layout/widgets/dial.py

```diff
@@ -0,0 +1,1096 @@
+# SPDX-FileCopyrightText: 2021 Kevin Matocha
+#
+# SPDX-License-Identifier: MIT
+"""
+
+`dial`
+================================================================================
+A dial gauge widget for displaying graphical information.
+
+* Author(s): Kevin Matocha
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+# pylint: disable=too-many-lines, too-many-instance-attributes, too-many-arguments
+# pylint: disable=too-many-locals
+
+
+import math
+import displayio
+import vectorio
+
+try:
+    import bitmaptools
+except NameError:
+    pass  # utilize the blit_rotate_scale function defined herein
+
+from adafruit_display_text import bitmap_label
+from adafruit_displayio_layout.widgets.widget import Widget
+
+
+class Dial(Widget):
+    """A dial widget.  The origin is set using ``x`` and ``y``.
+
+    :param int x: pixel position
+    :param int y: pixel position
+
+    :param int width: requested width in pixels
+    :param int height: requested height in pixels
+    :param int padding: keepout padding amount around the border, in pixels
+
+    :param float sweep_angle: dial rotation, in degrees, maximum value is 360 degrees
+    :param float start_angle: starting angle, in degrees.  Defaults
+     to `None` for symmetry along vertical axis
```

Comment:
I think we should mention in these `angle` parameter docstrings that 0 degrees is north or up or 12 o'clock (I'm not sure which term is best to convey the idea). Also a note that negative values will move to the left from there counter-clockwise.

Reply:
Updated docstring per your suggstion.